toolchain/check/testdata/facet/facet_assoc_const.carbon

```diff
@@ -247,15 +247,12 @@ library "[[@TEST_NAME]]";
 interface M { let X:! type; let Y:! type; let Z:! type; }
 
 // This fails because it resolves to `.X = .X` which is cyclical.
-// The value of .X and .Y becomes <error> but .Z is still valid.
```

Comment:
This stopped being the case in #5644, which I believe is correct. The phase of the facet type becomes an error when any error is encountered.